cluster-scope/overlays/prod/moc/smaug/groups/vault-osc-admins.yaml

```diff
@@ -5,3 +5,5 @@ metadata:
 users:
   - redmikhail
   - erikerlandson
+  - rynofinn
+  - mightynerderic
```

Comment:
@MightyNerdEric just noticed, can you correct the uppwer/lower casing here, as the openshift auth is case sensitivte.

Reply:
Done.